aptos-move/block-executor/src/view.rs

```diff
@@ -858,15 +870,17 @@ impl<'a, T: Transaction, X: Executable> ResourceGroupState<T> for SequentialStat
         group_key: &T::Key,
         resource_tag: &T::Tag,
         maybe_layout: Option<&MoveTypeLayout>,
-        patch_base_value: &dyn Fn(&T::Value, Option<&MoveTypeLayout>) -> anyhow::Result<T::Value>,
-    ) -> anyhow::Result<GroupReadResult> {
+        patch_base_value: &dyn Fn(&T::Value, Option<&MoveTypeLayout>) -> PartialVMResult<T::Value>,
+    ) -> PartialVMResult<GroupReadResult> {
         match self
             .unsync_map
             .fetch_group_tagged_data(group_key, resource_tag)
         {
             Ok(mut value) => {
                 // If we have a known layout, upgrade RawFromStorage value to Exchanged.
                 if let ValueWithLayout::RawFromStorage(v) = value {
+                    // TODO[agg_v2]: Can patching fail here, and should we do something about it directly?
+                    //               Probably yes, e.g. fallback?
```

Comment:
if transaction ends with DELAYED_FIELDS_CODE_INVARIANT_ERROR, when BlockSTM sees that - it does the sequential fallback

or you meant something else?

Reply:
Fixed error in patching, todo is not needed anymore.